### ISSUE TITLE
Theme color settings: color input not aligned with its label when label is too long

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4318]
Before:

![Screen Shot 2020-02-23 at 5 09 56 pm](https://user-images.githubusercontent.com/2423362/75104625-9468cf00-565f-11ea-8c59-ddf65d8f0e77.png)

After:

![Screen Shot 2020-02-23 at 5 10 05 pm](https://user-images.githubusercontent.com/2423362/75104629-9fbbfa80-565f-11ea-8b63-17f8a726ba0a.png)

---
PR by @klonos: https://github.com/backdrop/backdrop/pull/3084

**Comment by klonos** [comment]
...PR up for review, that simply sets the max width for labels for color inputs in this form to 80%.

**Comment by stpaultim** [comment]
Have we figured out the problem with the sandbox?

**Comment by klonos** [comment]
...I don't know what was wrong. I tried to close/reopen the PR a few times, but it was always the same thing. I have now closed the original PR in favor of a fresh: https://github.com/backdrop/backdrop/pull/3084

That new PR is off of the same branch, but the sandbox on that one works fine. Dunno 🤷‍♂

**Comment by klonos** [comment]
@stpaultim I have noticed that when the color input labels are spanning multiple lines, they seem to be crumbed up together:

![Screen Shot 2020-02-25 at 8 19 09 am](https://user-images.githubusercontent.com/2423362/75192207-e452ad80-57a7-11ea-82b2-c1aaf72771c5.png)

...do you think also address that here?

**Comment by klonos** [comment]
...fixed 🙂:

![Screen Shot 2020-02-25 at 8 55 49 am](https://user-images.githubusercontent.com/2423362/75194671-b02dbb80-57ac-11ea-9086-1753dd4df54d.png)

**Comment by klonos** [comment]
...fixed it a bit more (so that the color selectors are vertically aligned in the middle):

![Screen Shot 2020-02-25 at 9 06 18 am](https://user-images.githubusercontent.com/2423362/75195518-30a0ec00-57ae-11ea-80c0-a704e820230f.png)

**Comment by hosef** [comment]
I took a look at the code, and it looks good to me. I also checked out the color settings on the PR sandbox and it works as I expected it to.

**Comment by BWPanda** [comment]
@klonos A few minor changes needed to the code comments before this can be committed.

**Comment by klonos** [comment]
Thanks @hosef, @BWPanda 👍 

@BWPanda ...I've fixed the issue with the spaces (good catch; it slipped me), but I don't understand the other comment. Can you please elaborate?

**Comment by BWPanda** [comment]
@klonos Added comment to PR. If still not clear we can maybe talk tonight.

**Comment by klonos** [comment]
I've addressed the comment @BWPanda. Ready for another round when you have the time.

**Comment by BWPanda** [comment]
@klonos Thanks, looks good!

**Comment by klonos** [comment]
Hey @BWPanda if you were waiting on the CI tests to go green, I have finally managed to make that happen 😉 (not a problem with the PR, rather than with weirdness happening with ZenCI 😓) ...perhaps we can merge this in for 1.16.1 before we continue troubleshooting any CSS issues in #3930?

**Comment by BWPanda** [comment]
Thank you @klonos & @hosef! Merged https://github.com/backdrop/backdrop/pull/3084 into 1.x and 1.16.x.
